assets/app/js/controllers/conclusionDialog.js

```diff
@@ -283,7 +303,12 @@ class ConclusionDialog extends Controller {
           multiple: true,
           maximumSelectionLength: 1,
           placeholder: 'Enter Homepage URL',
-          tags: true
+          tags: true,
+          language: {
+            'noResults': function () {
+              return (`SCWB did not find any Homepage URL in the scan for ${conclusion.path}`);
```

Comment:
In some cases, this `conclusion.path` value is very long and is awkward to show in a small menu item. 

Instead of `${conclusion.path}`, we should have "the current path".